pkg/quota/v1/evaluator/core/services.go

```diff
@@ -144,12 +144,12 @@ func (p *serviceEvaluator) UsageStats(options quota.UsageStatsOptions) (quota.Us
 var _ quota.Evaluator = &serviceEvaluator{}
 
 //GetQuotaServiceType returns ServiceType if the service type is eligible to track against a quota, nor return ""
-func GetQuotaServiceType(service *v1.Service) v1.ServiceType {
+func GetQuotaServiceType(service *corev1.Service) corev1.ServiceType {
```

Comment:
This I did not expect.  @derekwaynecarr Does this break evaluation when received via admission?

Actually, are we sure this works for both the admission side and the reconciliation side today?

Reply:
isnt this pr changing both admission and reconciliation side to always use external?

Reply:
> isnt this pr changing both admission and reconciliation side to always use external?

@derekwaynecarr yeah. See the switch of the helper function to external as well.